pkg/osquery/runtime/history/history.go

```diff
@@ -21,15 +25,36 @@ func (c NoInstancesError) Error() string {
 	return "no osquery instances have been added to history"
 }
 
+// InitHistory loads the osquery instance history from bbolt DB if exists, sets up bucket if it does not
+func InitHistory(db *bbolt.DB) error {
+	currentHistory.Lock()
+	defer currentHistory.Unlock()
+
+	err := createBboltBucketIfNotExists(db)
+	if err != nil {
+		return err
+	}
+
+	currentHistory.db = db
+
+	err = currentHistory.load()
+
+	if err != nil {
```

Comment:
```suggestion
	if err =: currentHistory.load(); err != nil {
```

Nit... I wonder if we can lint for this.